agent/connect/testing_ca_test.go

```diff
@@ -35,29 +36,33 @@ func testCAAndLeaf(t *testing.T, keyType string, keyBits int) {
 		return
 	}
 
-	assert := assert.New(t)
+	require := require.New(t)
 
 	// Create the certs
 	ca := TestCAWithKeyType(t, nil, keyType, keyBits)
 	leaf, _ := TestLeaf(t, "web", ca)
 
 	// Create a temporary directory for storing the certs
 	td, err := ioutil.TempDir("", "consul")
-	assert.Nil(err)
-	defer os.RemoveAll(td)
+	require.NoError(err)
+	//defer os.RemoveAll(td)
+	t.Log(td)
```

Comment:
Did you meant to keep this logging in the PR?

Reply:
Nope good catch.